src/build.py

```diff
@@ -818,6 +821,7 @@ def LLVM():
   command = CMakeCommandNative([
       GetLLVMSrcDir('llvm'),
       '-DCMAKE_CXX_FLAGS=-Wno-nonportable-include-path',
+      '-DLLVM_ENABLE_LIBXML2=OFF',
```

Comment:
Does features of llvm does this disable?

Reply:
As mentioned in the commit message it appears to only be used for creating Windows manifests.